plugins/chat/assets/javascripts/discourse/components/chat-user-avatar.js

```diff
@@ -9,6 +9,7 @@ export default class ChatUserAvatar extends Component {
   user = null;
 
   avatarSize = "tiny";
+  showPresence = true;
```

Comment:
so we decided we didn't want to show presence in this case?

Reply:
Yes, see comment from Charlie further up this PR